.travis.yml

```diff
@@ -0,0 +1,68 @@
+stages:
+  - test
+  - commitlint
+  - name: release
+    if: branch = master AND type != pull_request
+
+sudo: required
+cache: bundler
+language: ruby
+
+services:
+  - docker
+
+# Make sure the instances listed below match up with
+# the `platforms` defined in `kitchen.yml`
+# NOTE: Please try to select up to six instances that add some meaningful
+#       testing of the formula's behaviour. If possible, try to refrain from
+#       the classical "chosing all the instances because I want to test on
+#       another/all distro/s" trap: it will just add time to the testing (see
+#       the discussion on #121).  As an example, the set chosen below covers
+#       the most used distros families, systemd and non-systemd and the latest
+#       three supported Saltstack versions with python2 and 3."
+#       As for `kitchen.yml`, that should still contain all of the platforms,
+#       to allow for comprehensive local testing
+#       Ref: https://github.com/saltstack-formulas/template-formula/issues/118
+#       Ref: https://github.com/saltstack-formulas/template-formula/issues/121
+env:
+  matrix:
+    - INSTANCE: repositories-debian-9-2019-2-py3
+    - INSTANCE: repositories-ubuntu-1804-2019-2-py3
+    - INSTANCE: preferences-debian-9-2019-2-py3
+    - INSTANCE: preferences-ubuntu-1804-2019-2-py3
```

Comment:
Any plan for `debian-8` and/or `ubuntu-1604`?

Reply:
Didn't see a valid reason to add them, as the files managed in this formula have not changed in a noticeable way.